lib/docker/image.js

```diff
@@ -79,6 +81,9 @@ class Image {
         args.push('--build-arg', `${name}=${value}`)
       }
     }
+    if (this.opts.target !== null) {
```

Comment:
the null check isn't necessary
```javascript
if (opts.target) args.push(...)
```

Reply:
Now that I think of this, it might be better to create an option for buildFlags that take arbitrary input flags for the build command. Then its pretty forward facing and the plugin doesn't need to change every time docker adds more things.

```javascript
{
  buildArgs: {GITHUB_TOKEN: null}
  buildFlags: {target: 'test-target'}
}
```